packages/example/pages/index.tsx

```diff
@@ -31,6 +32,21 @@ const defaultSummary: Summary = {
   balances: [],
 };
 
+type FeeTokens =
+  | CeloContract.GoldToken
```

Comment:
importing `CeloTokenContract` will give the same result but will mean we dont need to update this with each new stable token  that comes out

Reply:
Oh nice!